### PR TITLE
removed the 'unknows' string and cleand the proper error string prior…

### PR DESCRIPTION
**Pull request by danbar0kal**
… to throwing

### CODE REVIEW

**Comment by kaltura-hooks** [comment]
Hi @danbar0kal,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: api_v3/services/PartnerService.php [hunk 1]
```diff
@@ -139,7 +139,8 @@ public function registerAction( KalturaPartner $partner , $cmsPassword = "" , $t
 		{
 			KalturaLog::CRIT($ex);
 			// this assumes the partner name is unique - TODO - remove key from DB !
-			throw new KalturaAPIException( APIErrors::PARTNER_REGISTRATION_ERROR, 'Unknown error');
+			$exceptionMessage = (($pos = strpos($ex->getMessage(), 'KalturaPartner::')) !== FALSE) ? str_replace('KalturaPartner::', '', $ex->getMessage()) : $ex->getMessage();
```
Comment (MosheMaorKaltura) [inline api_v3/services/PartnerService.php:142]: The check is not really needed,
If you just remove 'KalturaPartner::' the results will be the same.
str_replace('KalturaPartner::', '',"TestKalturaPartner::123");
will yeild the same result as 
str_replace('KalturaPartner::', '',"Test123");

File: api_v3/services/PartnerService.php [hunk 2]
```diff
@@ -139,7 +139,8 @@ public function registerAction( KalturaPartner $partner , $cmsPassword = "" , $t
 		{
 			KalturaLog::CRIT($ex);
 			// this assumes the partner name is unique - TODO - remove key from DB !
-			throw new KalturaAPIException( APIErrors::PARTNER_REGISTRATION_ERROR, 'Unknown error');
+			$exceptionMessage = (str_replace('KalturaPartner::', '', $ex->getMessage()));
```
Comment (inbal-ben-david) [inline api_v3/services/PartnerService.php:142]: keep the original exception catch as it was and instead add a catch for kCoreException that expose the error massage

Comment (danbar0kal) [reply api_v3/services/PartnerService.php:141]: done

**Comment by inbal-ben-david** [comment]
reviewd

**Pull request merged**